### DISCUSSION TITLE
Can't figure out how to completely disable notifications... is it impossible?

### DISCUSSION THREAD

**Discussion by joaociocca** [kovidgoyal/kitty#7639] [General]
So. Title. That's it. I've been searching around this all day, tried `update_check_interval·0`, tried `notify_on_cmd_finish·never`, but it keeps popping up!

![image](https://github.com/user-attachments/assets/fae57fc4-9120-4b9f-9a6b-ae216aaefffd)

**Comment by kovidgoyal** [comment]
https://github.com/nushell/nushell/issues/13092

**Reply by joaociocca** [reply]
> Since this is an option that is
> 
> Not turned on by default, and

but I did not enable this? I just installed kitty and started using and the notification keeps popping up?

**Comment by kovidgoyal** [comment]
On Wed, Jul 17, 2024 at 09:42:45PM -0700, João Ciocca wrote:
> > Since this is an option that is
> >
> > Not turned on by default, and
>
> but I did not enable this? I just installed kitty and started using and the notification keeps popping up?

The notification is coming from something your are running inside kitty.
Figure out whatever that is and stop it from doing so.

**Reply by joaociocca** [reply]
~~This is too weird. When I open kitty "normally", I get these notifications when it opens up and again for each command. But if I open it from Dolphin's "open terminal here" functionality, the notifications don't show up...?~~ stupid, of course Dolphin's integration doesn't trigger notifications, it's opening Konsole instead of kitty...

ok, let's take it from the top... looking at the documentation https://sw.kovidgoyal.net/kitty/desktop-notifications/ I should be looking for something like `printf '\x1b]99;;Hello world\x1b\\'`, but which is outputting "9;${HOME}"...

`grep -Erni "printf ['\"]\\\\x1b"` should do the job, right?

I found it only on a "done.fish" file, but I'm using zsh, not fish?

```
.config/fish/conf.d/done.fish:256:                printf "\x1b]99;i=done:d=0;$title\x1b\\"
.config/fish/conf.d/done.fish:257:                printf "\x1b]99;i=done:d=1:p=body;$message\x1b\\"
```

but if I switch to fish, the notifications disappear. 

When I comment out the execution of my personal script, the notifications also disappear... but there's absolutely no `pintf '\x1b` in there... is there any other thing that could trigger the notification?

ok, this seems to be related to [oh-my-posh's Kushal theme](https://github.com/JanDeDobbeleer/oh-my-posh/blob/main/themes/kushal.omp.json). When I load a different theme, there's no notifications.

Tried another terrminal, "Contour", and in this one I get errors on its console:

```
[2024-07-18 17:49:39.119862735.119862] [vt.parser] Unknown VT sequence: OSC 9 9;/dev/home
```

Ok, Oh-My-Posh creator helped me figure out [what was happening](https://github.com/JanDeDobbeleer/oh-my-posh/discussions/5303#discussioncomment-10088462).
